README.md

```diff
@@ -25,14 +25,18 @@ and [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/vers
 1. Fork and clone the Git repo:
    `git clone https://github.com/<your_username>/python-zulip-api.git`
 
-2. Make sure you have [pip](https://pip.pypa.io/en/stable/installing/)
+2. Setup upstream and fetch upstream/master (Required for Gitlint):
+   - `git remote add upstream https://github.com/zulip/python-zulip-api.git`
+   - `git fetch upstream master`
+
+3. Make sure you have [pip](https://pip.pypa.io/en/stable/installing/)
    and [virtualenv](https://virtualenv.pypa.io/en/stable/installation.html)
    installed.
```

Comment:
Should this documentation be adjusted now that we are using the `venv` module, to just request `pip`?